src/LinearMap.jl

```diff
@@ -37,6 +37,7 @@ Return the linear map of a convex set.
 If the matrix is null, a `ZeroSet` is returned; otherwise a lazy linear map.
```

Comment:
Is "null" a common term?

Reply:
as a matrix whose entries are all zero, see the wikipedia article [Zero matrix](https://en.wikipedia.org/wiki/Zero_matrix). so i could use that, or "If all the entries of the matrix are zero, ..."

Reply:
I am fine with "null".